File: Mardis.Engine.Web/wwwroot/js/viewModel/PollViewModel.js
//
function PollViewModel() {
    var self = this;

    self.IdSection = ko.observable();
    self.Title = ko.observable();
    self.ItemsQuestion = ko.observableArray();
    self.Weight = ko.observable();

}

//
function QuestionViewModel() {
    var self = this;

    self.IdQuestion = ko.observable();
    self.Title = ko.observable();
    self.Weight = ko.observable();
    self.IdTypePoll = ko.observable();
    self.ItemsAnswer = ko.observableArray();
    self.HasPhotos = ko.observable();
    self.CountPhotos = ko.observable();

}

//
function AnswerViewModel() {
    var self = this;

    self.IdAnswer = ko.observable();
    self.Title = ko.observable();
    self.Weight = ko.observable();
}

File: Mardis.Engine.Web.ViewModel/PollViewModel.cs
using System;
using System.Collections.Generic;

namespace Mardis.Engine.Web.ViewModel
{
    /// <summary>
    /// 
    /// </summary>
    public class PollViewModel
    {
        public PollViewModel()
        {
            ItemsQuestion = new List<QuestionPollViewModel>();
        }

        public Guid IdSection { get; set; }
        public string Title { get; set; }
        public string Weight { get; set; }

        public bool IsDynamic { get; set; }

       public List<QuestionPollViewModel> ItemsQuestion { get; set; }

    }

    /// <summary>
    /// 
    /// </summary>
    public class QuestionPollViewModel
    {
        public QuestionPollViewModel()
        {
            ItemsAnswer = new List<AnswerPollViewModel>();
        }

        public Guid IdQuestion { get; set; }

        public string Title { get; set; }

        public Guid IdTypePoll { get; set; }

        public string Weight { get; set; }

        public string HasPhotos { get; set; }

        public string CountPhotos { get; set; }

        public List<AnswerPollViewModel> ItemsAnswer { get; set; }

    }

    /// <summary>
    /// 
    /// </summary>
    public class AnswerPollViewModel
    {
        public Guid IdAnswer { get; set; }
        public string Title { get; set; }
        public string Weight { get; set; }

    }

}
